types/mydecimal.go

```diff
@@ -1991,7 +1990,6 @@ func doDivMod(from1, from2, to, mod *MyDecimal, fracIncr int) error {
 			idxTo++
 			digitsIntTo++
 		}
-		digitsIntTo++
```

Comment:
Why this line removed?

Reply:
Since it is unused anymore. Is that a kind of bug?